packages/common/src/ceramic-api.ts

```diff
@@ -146,8 +130,8 @@ export interface AdminApi {
 /**
  * Describes Ceramic node API
  */
-export interface CeramicApi extends CeramicSigner {
-  // loggerProvider: LoggerProvider; // TODO uncomment once logger is available on http-client
+export interface CeramicApi {
```

Comment:
it's not really immediately clear to me why the methods that remain in this interface are still here while so many of the others have been removed.  Can you add some more comments explaining what the purpose of this interface is now going forward, and how it differs from StreamReaderWriter?

Reply:
Actually, is this interface even used anywhere anymore?  Can it be deleted?